react/features/dynamic-branding/functions.any.js

```diff
@@ -6,11 +6,21 @@ import { loadConfig } from '../base/lib-jitsi-meet';
  * Extracts the fqn part from a path, where fqn represents
  * tenant/roomName.
  *
- * @param {string} path - The URL path.
+ * @param {Object} state - A redux state.
  * @returns {string}
  */
-export function extractFqnFromPath() {
-    const parts = window.location.pathname.split('/');
+export function extractFqnFromPath(state?: Object) {
+    let pathname;
+
+    if (window.location.pathname) {
+        pathname = window.location.pathname;
+    } else if (state['features/base/connection']) {
```

Comment:
since state is optional here, you should check for truthiness of `state` before doing this evaluation

Reply:
done